src/Entity/Slot.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace Drupal\acquia_lift\Entity;
+
+use Acquia\LiftClient\Entity\Visibility;
+use Drupal\acquia_lift\AcquiaLiftException;
+use Drupal\acquia_lift\SlotInterface;
+use Drupal\Core\Config\Entity\ConfigEntityBase;
+use Drupal\Core\Entity\EntityStorageInterface;
+
+/**
+ * Defines the Drupal created slots.
+ *
+ * @ConfigEntityType(
+ *   id = "acquia_lift_slot",
+ *   label = @Translation("Slot"),
+ *   label_singular = @Translation("slot"),
+ *   label_plural = @Translation("slots"),
+ *   label_count = @PluralTranslation(
+ *     singular = "@count slot",
+ *     plural = "@count slots",
+ *   ),
+ *   handlers = {
+ *     "storage" = "Drupal\Core\Config\Entity\ConfigEntityStorage",
+ *     "list_builder" = "Drupal\acquia_lift\Entity\Controller\SlotListBuilder",
+ *     "form" = {
+ *       "default" = "Drupal\acquia_lift\Form\SlotForm",
+ *       "edit" = "Drupal\acquia_lift\Form\SlotForm",
+ *       "delete" = "Drupal\acquia_lift\Form\SlotDeleteConfirmForm",
+ *       "disable" = "Drupal\acquia_lift\Form\SlotDisableConfirmForm",
+ *       "enable" = "Drupal\acquia_lift\Form\SlotEnableConfirmForm",
+ *     },
+ *   },
+ *   admin_permission = "administer acquia_lift",
+ *   config_prefix = "slot",
+ *   entity_keys = {
+ *     "id" = "id",
+ *     "label" = "label",
+ *     "uuid" = "uuid",
+ *     "status" = "status",
+ *   },
```

Comment:
entity keys - mapping of the main entity keys to the entity properties we defined. For instance, when we call the label() method on the entity object, it will call the label property. I don't think any key should be in here if it ain't different.